### ISSUE TITLE
Need ontap module to automate volume efficiency start command.

### ISSUE THREAD

**Issue by autocoder10** [ansible-collections/netapp#84]
##### SUMMARY
We are looking to automate the ontap volume efficiency start start command
##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
na_ontap

##### ADDITIONAL INFORMATION

```
CLI command:
volume efficiency start -vserver <vserver_name> -volume <volume_name> -scan-old-data true

```

**Comment by carchi8py** [comment]
We recently released a volume Efficiency Module in our Feb 2021 release.  Did this module not cover what you were looking for

https://github.com/ansible-collections/netapp/blob/master/ansible_collections/netapp/ontap/plugins/modules/na_ontap_volume_efficiency.py

**Comment by autocoder10** [comment]
I have reviewed that module and I see only these functions "Enable, modify or disable volume efficiency". So need the volume efficiency start one.

**Comment by autocoder10** [comment]
@carchi8py : Can you please provide any update on this request.

**Comment by carchi8py** [comment]
@autocoder10 Sorry for the delay (we used to have github message coming directly to our slack instance but redhat appear to turn this off so we haven't been seeing these until we manually looked). give me a day to look in to this.

**Comment by autocoder10** [comment]
@carchi8py : Can you please provide any update on this request.

**Comment by lonico** [comment]
Fixed in ONTAP collection 21.4.0

na_ontap_volume_efficiency - updated to now allow for storage efficiency start and storage efficiency stop.
